content/zh/docs/languages/java/_index.md

```diff
@@ -7,8 +7,9 @@ aliases: [/java, /java/metrics, /java/tracing]
 cascade:
   vers:
     instrumentation: 2.7.0
-    otel: 1.41.0
-    semconv: 1.26.0
+    otel: 1.42.1
+    contrib: 1.38.0
+    semconv: 1.27.0
```

Comment:
@open-telemetry/docs-maintainers - a reminder that an `en` PR shouldn't be updating non-`en` pages.